templates/zib-Problem.json

```diff
@@ -0,0 +1,64 @@
+{
+  "resourceType": "Condition",
+  "id": "{{resource_id}}",
+  "meta": {
+    "profile": [
+      "http://nictiz.nl/fhir/StructureDefinition/zib-Problem"
+    ]
+  },
+  "text": {
+    "status": "extensions",
+    "div": "{{html_description}}"
+  },
+  "{% if clinical_status_code %}"
```

Comment:
These if statements break the json. And I am not sure if this will be parsed correctly with Jinja2?